pkg/apis/machineconfiguration.openshift.io/v1/helpers.go

```diff
@@ -196,3 +204,44 @@ func IsControllerConfigCompleted(ccName string, ccGetter func(string) (*Controll
 	}
 	return fmt.Errorf("ControllerConfig has not completed: completed(%v) running(%v) failing(%v)", completed, running, failing)
 }
+
+// ProxyInspectTest inspects the proxyTestImage to test if the http proxy is valid.
+func ProxyInspectTest(proxy *configv1.ProxyStatus) error {
```

Comment:
Generally things with Test in the name are tests. Can you rename to show what this check is doing?